### PR TITLE
Isis CTX Driver Fixes

### PR DESCRIPTION
**Pull request by acpaquette**
Changes to get an ISIS Ctx driver working for comparisons with ISIS

### CODE REVIEW

File: ale/drivers/mro_drivers.py [hunk 1]
```diff
@@ -39,27 +42,25 @@ def spacecraft_id(self):
         : int
           Naif ID code for the spacecraft
         """
-        return "N/A"
+        return "-74"
```
Comment (Kelvinrr) [inline ale/drivers/mro_drivers.py]: Is this not in the kernels?

File: ale/drivers/mro_drivers.py [hunk 2]
```diff
@@ -39,27 +42,25 @@ def spacecraft_id(self):
...
+    def sensor_model_version(self):
         """
         Returns
         -------
         : int
-          Naif ID code for the instrument
+          ISIS sensor model version
         """
-        return int(self.label["IsisCube"]["Kernels"]["NaifFrameCode"])
+        return 1
```
Comment (kberryUSGS) [inline ale/drivers/mro_drivers.py]: I'm not sure how this is used by CSM, so perhaps it doesn't matter, but couldn't this come from ["IsisCube"]["Kernels"]["CameraVersion"] ?

File: ale/drivers/mro_drivers.py (hunk 1 above)
Comment (jessemapel) [reply ale/drivers/mro_drivers.py]: The `spice.bods2c(self.spacecraft_name)` call in data_naif should cover this if it's in the kernels as @Kelvinrr pointed out. Is the spacecraft_name not returning the correct value?

File: ale/drivers/mro_drivers.py (hunk 2 above)
Comment (jessemapel) [reply ale/drivers/mro_drivers.py]: This is not used at all by CSM, but this value should be coming in off of that location. That code probably needs to be added to data_isis

**Comment by acpaquette** [comment]
@Kelvinrr This doesn't really fix the comparison to ISIS when outputting a label from a PDS image but makes it so I could compare the isis directly to a label generated from a cube with the same positions, velocities, etc.

File: ale/drivers/mro_drivers.py (hunk 1 above)
Comment (acpaquette) [reply ale/drivers/mro_drivers.py]: Does this work on the ISIS side? I thought we didn't load any kernels?

Comment (jessemapel) [reply ale/drivers/mro_drivers.py]: No this doesn't work without kernels sorry!

Comment (acpaquette) [reply ale/drivers/mro_drivers.py]: I think we could get this from one of the frame codes but it would be a little hacky

**Pull request merged**